editor/components/no-blocks/index.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Maybe this should be just a file inside inserter `inserter/no-blocks.js` as I don't expect it to be used elsewhere

Reply:
True currently we don't see it used somewhere else & makes sense moving it to "Inserter". Was confused where to put thank you :)